packages/mask/src/plugins/MaskBox/helpers/formatCountdown.ts

```diff
@@ -13,20 +13,16 @@ const units = [
 export function formatCountdown(countdown: number) {
     if (countdown <= 0) return ''
 
-    const amounts: number[] = []
+    const segs: string[] = []
 
-    ;[msPerDay, msPerHour, msPerMinute, msPerSecond].reduce((accumulator, x) => {
-        amounts.push(Math.floor(accumulator / x))
+    ;[msPerDay, msPerHour, msPerMinute, msPerSecond].reduce((accumulator, x, index, list) => {
+        const isLast = index === list.length - 1
+        const value = isLast ? Math.ceil(accumulator / x) : Math.floor(accumulator / x)
+        if (segs.length !== 0 || value !== 0) {
+            segs.push(`${value} ${units[index][value === 1 ? 0 : 1]}`)
+        }
```

Comment:
Show `1 hour 0 minute 0 second` instead of `1 hour`

Reply:
why not use https://date-fns.org/v2.27.0/docs/intervalToDuration?

Reply:
e.g:
```
formatDuration(intervalToDuration({
    start: new Date(0),
    end: new Date(countdown * 1000)
}))
```